examples/plot_map_in_3d.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
Should we not skip this example until we work around the lack of a display server?

Reply:
We can leave it in and not execute it right?

Reply:
It executes it right now.